File: app/src/main/java/ru/roscha_akademii/medialib/viewvideo/view/StubExoPlayerEventListener.java
package ru.roscha_akademii.medialib.viewvideo.view;

import android.util.Log;

import com.google.android.exoplayer2.ExoPlaybackException;
import com.google.android.exoplayer2.ExoPlayer;
import com.google.android.exoplayer2.Timeline;

public class StubExoPlayerEventListener implements ExoPlayer.EventListener {
    @Override
    public void onLoadingChanged(boolean isLoading) {
        Log.d("happy", "event onLoadingChanged " + isLoading);
    }

    @Override
    public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
        Log.d("happy", "event onPlayerStateChanged " + playWhenReady + " " + playbackState);
    }

    @Override
    public void onTimelineChanged(Timeline timeline, Object manifest) {
        Log.d("happy", "event onTimelineChanged");
    }

    @Override
    public void onPlayerError(ExoPlaybackException error) {
        Log.d("happy", "event onPlayerError " + error.getMessage());
    }

    @Override
    public void onPositionDiscontinuity() {
        Log.d("happy", "event onPositionDiscontinuity");
    }
}

File: app/src/main/java/ru/roscha_akademii/medialib/video/playvideo/viewvideo/tools/StubExoPlayerEventListener.kt
package ru.roscha_akademii.medialib.video.playvideo.viewvideo.tools

import android.util.Log

import com.google.android.exoplayer2.ExoPlaybackException
import com.google.android.exoplayer2.ExoPlayer
import com.google.android.exoplayer2.Timeline

open class StubExoPlayerEventListener : ExoPlayer.EventListener {
    override fun onLoadingChanged(isLoading: Boolean) {
        Log.d("happy", "event onLoadingChanged " + isLoading)
    }

    override fun onPlayerStateChanged(playWhenReady: Boolean, playbackState: Int) {
        Log.d("happy", "event onPlayerStateChanged $playWhenReady $playbackState")
    }

    override fun onTimelineChanged(timeline: Timeline?, manifest: Any?) {
        Log.d("happy", "event onTimelineChanged")
    }

    override fun onPlayerError(error: ExoPlaybackException) {
        Log.d("happy", "event onPlayerError " + error.message)
    }

    override fun onPositionDiscontinuity() {
        Log.d("happy", "event onPositionDiscontinuity")
    }
}
